db/migrate/20190904104533_create_user_security_keys.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class CreateUserSecurityKeys < ActiveRecord::Migration[5.2]
+  def up
+    create_table :user_security_keys do |t|
+      t.references :user, null: false, index: true, foreign_key: true
+      t.string :credential_id, null: false
+      t.string :public_key, null: false, index: true
+      t.integer :factor_type, null: false, default: 0, index: true
+      t.boolean :enabled, null: false, default: true
```

Comment:
There is a scope for `enabled` above - thoughts on adding that as part of an index?

Reply:
You're right. Will add this and some model scopes to bring more into line with `UserSecondFactor`